integration_tests/commands/setup.go

```diff
@@ -76,7 +83,12 @@ func FireCommand(conn net.Conn, cmd string) interface{} {
 		if err == io.EOF {
 			return nil
 		}
-		log.Fatalf("error %s while firing command: %s", err, cmd)
+		slog.Error(
+			"error while firing command",
+			slog.Any("error", err),
+			slog.String("command", cmd),
+		)
+		os.Exit(1)
```

Comment:
Shall we write our own function wrapper like `func LogErrorFatal(l *slog.Logger, ..)` and use it instead of calling `os.Exit(1)`?
May be we can do this as part of package implementaion.

Reply:
So my reasoning behind not doing it was this:
1. it would be a bit awkward/unintuitive to use the logger if all methods are from slog, but one of them is package level. slog doesn't offer it a `Fatal` method
2. Fatal errors should explicitly exit, since they are also rarer and should be generally avoided. The logger causing program to exit is a bit of weird behaviour.
3. `os.Exit` gives us explicit control over status code we want to exit with. Might be useful when other applications depend on the status code from the db.


Let me know if any of these makes sense, if you still think it's better to wrap it, should we wrap all methods (kinda defeats the purpose of using slog IMO) or just this one function?